packages/maskbook/src/env.d.ts

```diff
@@ -125,3 +125,8 @@ declare module '*.jpg' {
     const content: string
     export default content
 }
+
+declare module '*.svg' {
```

Comment:
Hello please remove those. We're using `new URL(..., import.meta.url)` to load resources.